File: __tests__/fixtures/output/python/python3/cookies.py
import http.client

conn = http.client.HTTPSConnection("httpbin.org")

headers = { 'cookie': "foo=bar; bar=baz" }

conn.request("GET", "/cookies", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: src/targets/javascript/jquery/fixtures/cookies.js
const settings = {
  async: true,
  crossDomain: true,
  url: 'https://httpbin.org/cookies',
  method: 'GET',
  headers: {
    cookie: 'foo=bar; bar=baz'
  }
};

$.ajax(settings).done(res => {
  console.log(res);
});